tests/end_to_end/L1L2Messaging/test_messaging.py

```diff
@@ -114,38 +154,28 @@ async def test_should_fail_unauthorized_message_sender(
 @pytest.mark.slow
 @pytest.mark.asyncio(scope="module")
 class TestL1ToL2Messages:
-    async def test_should_increment_counter_on_l2(self, message_app_l1, message_app_l2):
+    async def test_should_increment_counter_on_l2(
+        self, message_app_l1, message_app_l2, wait_for_l2_messaging
+    ):
         msg_counter_before = await message_app_l2.receivedMessagesCounter()
         increment_value = 1
+        current_l2_block = await RPC_CLIENT.get_block_number()
         message_app_l1.increaseL2AppCounter(
             message_app_l2.address, value=increment_value
         )
-        time.sleep(4)
+        await wait_for_l2_messaging(current_l2_block, message_app_l2.address)
         msg_counter_after = await message_app_l2.receivedMessagesCounter()
         assert msg_counter_after == msg_counter_before + increment_value
 
-    async def test_should_apply_alias_from_l1(self, message_app_l1, message_app_l2):
+    async def test_should_apply_alias_from_l1(
+        self, message_app_l1, message_app_l2, wait_for_l2_messaging
+    ):
         msg_counter_before = await message_app_l2.receivedMessagesCounter()
         increment_value = 1
+        current_l2_block = await RPC_CLIENT.get_block_number()
         message_app_l1.increaseL2AppCounterFromCounterPartOnly(
             message_app_l2.address, value=increment_value
         )
-        time.sleep(4)
+        await wait_for_l2_messaging(current_l2_block, message_app_l2.address)
         msg_counter_after = await message_app_l2.receivedMessagesCounter()
         assert msg_counter_after == msg_counter_before + increment_value
-
-    async def test_should_fail_unauthorized_message_sender(
```

Comment:
removing this one?

Reply:
added as unit test instead to avoid updating kakarot real values in staging